tools/metadatastore-upgrade/README.md

```diff
@@ -0,0 +1,34 @@
+# MetadataStore Upgrade Tool
+
+Upgrade tool provides a mechanism for KFP users to upgrade MetadataStore component in their KFP cluster. A MetadatatStore upgrade is composed of two aspects:
+* Upgrading the image used in `metadata-deployment` K8s Deployment. The user is expected to provide an image-tag for this upgrade.
```

Comment:
Good to provide a link here to the canonical definition? Maybe standalone manifests?

Reply:
Done.